ci/Dockerfile

```diff
@@ -20,8 +20,9 @@ RUN wget -q https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key &&
     apt-get install -y cf-cli
 
 # Install ruby dependencies
-RUN mkdir -p "$CI_PATH" && \
-    gem install bundler && \
+RUN gem update --system && \
+    mkdir -p "$CI_PATH" && \
+    gem install bundler -v 2.1.4 && \
```

Comment:
Would this be more future-proof to install the Bundler version based on the "BUNDLED WITH" content in Gemfile.lock, as described in the bug that you linked to, e.g.:
```
gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)"
```